packet/flatcar-linux/kubernetes/calico/host-endpoint-controller.yaml

```diff
@@ -0,0 +1,72 @@
+# manifests for component calico-hostendpoint-controller
```

Comment:
Should we not include this as a default in `terraform-render-bootkube`, instead of crowding this repo with mix of some manifests and some cluster creating code? Because this does not need any change at and works fine out of the box?


But this is not relevant to other deployment and cause unknown issues, so I am ambivalent about this.

Reply:
I think it makes sense to have it on Lokomotive since it is related to the network policy manifests we added

Reply:
cool, if folks agree that is the right thing to do then. Please go ahead.

I just wanted to have a consensus on what goes where.

Reply:
It's specific for Packet, we don't want to run this on other platforms, also (maybe on baremetal, but only maybe it really depends on the setup. I think probably is not even needed there either)

Are you ok with having this here? It makes me wonder a little bit too. But I think is okay... :-D

Reply:
I am okay, just wanted to make sure we cognizant about this and moving forward. So all good @kosyfrances :+1: